src/ucp/api/ucp.h

```diff
@@ -679,10 +682,40 @@ typedef struct ucp_ep_params {
      * Handler to process transport level failure.
      */
     ucp_err_handler_t       err_handler;
+
+    /**
+     * User data associated with an endpoint. See @ref ucp_ep_poll_t.
+     * @a user_data redefines @ref ucp_err_handler_t::arg if set both
```

Comment:
@evgeny-leksikov is it possible to make documentation more clear for this point?